x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -152,18 +153,80 @@ export async function installKibanaAssets(options: {
     managed: true,
   });
 
-  await makeManagedIndexPatternsGlobal(savedObjectsClient);
+  await makeManagedIndexPatternsGlobal(soClient);
 
   const installedAssets = await installKibanaSavedObjects({
     logger,
     savedObjectsImporter,
-    kibanaAssets: assetsToInstall,
+    kibanaAssets: assetsToInstall.filter((asset) => asset.type !== KibanaSavedObjectType.slo),
   });
 
+  const sloAssets = assetsToInstall.filter((asset) => asset.type === KibanaSavedObjectType.slo);
+
+  await installSLOAssets({ sloAssets, sloClient, soClient, esClient, spaceId });
+
   return installedAssets;
 }
 
+export async function installSLOAssets({
+  sloClient,
+  soClient,
+  esClient,
+  spaceId,
+}: {
+  sloAssets: ArchiveAsset[];
+  sloClient: SLOClient;
+  soClient: SavedObjectsClientContract;
+  esClient: ElasticsearchClient;
+  spaceId: string;
+}) {
+  // if (!sloAssets.length) {
+  //   return;
+  // }
+
+  const sloAssets = [
```

Comment:
@shahzad31 Is this here temporarily just for testing purposes? I guess this will be somehow read from the [integration](https://github.com/elastic/integrations/pull/10269), right?